src/lib/runtime/sandbox/Web3FunctionThreadSandbox.spec.ts

```diff
@@ -0,0 +1,198 @@
+import path from "node:path";
+
+import { Web3FunctionVersion } from "../../types";
+import { Web3FunctionThreadSandbox } from "./Web3FunctionThreadSandbox";
+
+describe("Web3FunctionThreadSandbox", () => {
+  const SCRIPT_FOLDER = path.join(
+    process.cwd(),
+    "src/lib/runtime/sandbox/__test__/"
+  );
+
+  test("pass correct arguments to runner v1", async () => {
```

Comment:
is it v1 or v2?

the test is using: `Web3FunctionVersion.V2_0_0`

Reply:
fixed in ccaf9abedf6194822325974f3a756c6cd8d2c1db